pkg/process/checks/remote_sysprobe_util_test.go

```diff
@@ -0,0 +1,103 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Optional nit: rename file to `sysprobe_stats_test.go`, as we don't use the RemoteSysProbeUtil.